frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -314,6 +349,15 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({
       <p className="control-label help-block" id="nodes-selected">
         {selectedNodesCnt} node(s) selected
       </p>
+      {/* double negation to convert to boolean, else in case of 0 length, 0 is gettin shown */}
+      {!!nodesWarningMsg.length && (
```

Comment:
```suggestion
      {nodesWarningMsg.length >=1 && (
```

Reply:
Can you please explain the reason behind this

Reply:
Looks cleaner to me. But I think just `nodesWarningMsg` should be good enough

Reply:
Looks cleaner to me for the above suggestion but I think here we can do this: 
```suggestion
      {nodesWarningMsg && (
```

Reply:
@cloudbehl  in case of 0 length, 0 will be shown with this logic. I have also added this comment in the code.